From f4f2f19260ec41b1c6f0878ac3f17edef384ba92 Mon Sep 17 00:00:00 2001
From: Jia Hongtao <hongtao.jia@freescale.com>
Date: Wed, 6 Aug 2014 10:47:54 +0800
Subject: [PATCH 001/129] Add LTC2945 node to ls1021a-twr.dts

Signed-off-by: Jia Hongtao <hongtao.jia@freescale.com>
Change-Id: I08008c3dcd2b85b0c54b9f9ee939287f57745517
Reviewed-on: http://git.am.freescale.net:8181/19641
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Jingchang Lu <jingchang.lu@freescale.com>
Reviewed-by: Zhengxiong Jin <Jason.Jin@freescale.com>
[Xulin: Original patch taken from
Freescale-Linux-SDK-for-LS1021A-IOT-Rev2-v0.4-SOURCE-20150907-yocto.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/boot/dts/ls1021a-twr.dts |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/ls1021a-twr.dts b/arch/arm/boot/dts/ls1021a-twr.dts
index a2c591e..dbe5aec 100644
--- a/arch/arm/boot/dts/ls1021a-twr.dts
+++ b/arch/arm/boot/dts/ls1021a-twr.dts
@@ -81,6 +81,13 @@
 	status = "okay";
 };
 
+&i2c2 {
+	status = "okay";
+	monitor: ltc2945@67 {
+		reg = <0x67>;
+	};
+};
+
 &ifc {
 	#address-cells = <2>;
 	#size-cells = <1>;
-- 
1.7.5.4

